src/common/constants/cssScale.ts

```diff
@@ -1,14 +1,15 @@
 export const cssScale = {
   c0: "0",
-  c1: "4px",
-  c2: "8px",
-  c3: "12px",
-  c4: "16px",
-  c5: "20px",
-  c6: "24px",
-  c7: "28px",
-  c8: "32px",
-  c9: "36px",
-  c10: "40px",
-  c11: "44px",
+  c1: "2px",
+  c2: "4px",
+  c3: "8px",
+  c4: "12px",
+  c5: "16px",
+  c6: "20px",
+  c7: "24px",
+  c8: "28px",
+  c9: "32px",
+  c10: "36px",
+  c11: "40px",
+  c12: "44px",
```

Comment:
You'd need to update the previous usage sites of these values.

Reply:
Reverted the change.